### ISSUE TITLE
Partials don't support path aliases

### ISSUE THREAD

**Issue by cedx** [cedx/yii2-mustache#8]
The documentation says that we can use path aliases for partials:

```html
{{> @app/views/header.mustache }}
```

But this triggers a `yii\base\InvalidCallException` in `yii\mustache\Loader` class:

```
The view file "[...]/views/header.mustache" does not exist.
```

**Comment by cedx** [comment]
Yes, the `Loader` class provides its own implementation of [`yii\base\View::findViewFile`](https://www.yiiframework.com/doc/api/2.0/yii-base-view#findViewFile()-detail) method, and this implementation does not handle path aliases.

**Comment by cedx** [comment]
Fixed in version 10.0.0 (released yesterday).
